package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@types/mocha": "^5.2.5",
     "@types/node": "^12.7.2",
+    "commander": "^5.0.0",
```

Comment:
you shouldn't need this actually...if you look at the package.json of icon-magic/cli, commander is already included there.

Reply:
but this looks v good!! thanks!

Reply:
ok, removed in latest commit, Ty